sections/designRules.md

```diff
@@ -135,11 +135,13 @@ Although the REST architectural style does not impose a specific protocol, REST
    <dl>
       <dt>Statement</dt>
       <dd>
-         Resources MUST be retrieved or manipulated using standard HTTP methods (GET/POST/PUT/PATCH/DELETE).
+         Resources MUST be retrieved or manipulated using standard HTTP methods (GET/POST/PUT/PATCH/DELETE). Resources MAY support HTTP methods that provide information without manipulating the underlying resources (including but not limited to HEAD/CONNECT/OPTIONS/TRACE).
       </dd>
       <dt>Rationale</dt>
       <dd>
          The HTTP specifications offer a set of standard methods, where every method is designed with explicit semantics. Adhering to the HTTP specification is crucial, since HTTP clients and middleware applications rely on standardized characteristics.
+         <p>According to <a href="https://www.rfc-editor.org/rfc/rfc9110#name-overview">RFC 9110 9.1</a> the <code>GET</code> and <code>HEAD</code> HTTP methods MUST be supported by the server, all other methods are optional.
```

Comment:
Wat wordt hier bedoeld met `MUST be supported by the server`? Op welke resources? Bij operation resources kan het zijn dat er geen `GET` (en dus ook `HEAD`) operatie bestaat.

Reply:
```suggestion
```